.coderabbit.yml

```
@@ -0,0 +1,61 @@
+reviews:
+  path_instructions:
+    - path: "**/*.go"
+      instructions: |
+       
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)